src/test/java/org/folio/rest/impl/PurchaseOrdersApiTest.java

```diff
@@ -628,6 +628,7 @@ public void testPutOrdersByIdPEMixFormat() {
     CompositePurchaseOrder reqData = getMockAsJson(PE_MIX_PATH).mapTo(CompositePurchaseOrder.class);
 
     reqData.setId(ID_FOR_PRINT_MONOGRAPH_ORDER);
+    reqData.setReEncumber(null);
```

Comment:
Shouldn't this be set to `false`?

Reply:
I'm checking that nothing is failed if reEncumber is empty